src/ui/CSimpleEditBoxScript.cpp

```diff
@@ -129,7 +129,19 @@ int32_t CSimpleEditBox_Insert(lua_State* L) {
 }
 
 int32_t CSimpleEditBox_SetText(lua_State* L) {
-    WHOA_UNIMPLEMENTED(0);
+    int32_t type = CSimpleEditBox::GetObjectType();
+    auto editBox = static_cast<CSimpleEditBox*>(FrameScript_GetObjectThis(L, type));
+
+    if (!lua_isstring(L, 2)) {
+        return luaL_error(L, "Usage: %s:SetText(\"text\")", editBox->GetDisplayName());
+    }
+
+    const char* lua_tainted = nullptr;
```

Comment:
I figured that when we implement lua_tainted properly, we'll get a compile error here and we'll remove the definition. Let me know if you want me to change this.

Reply:
`lua_tainted` is a global, but there won't be a compile error simply because it exists. Rather, this `lua_tainted` will end up being a shadow variable scoped to this function.

Let's just stick to `// TODO lua_tained` for now, and pass `nullptr` as the final argument in `CSimpleEditBox::SetText` below.